feature/achievements/src/main/java/io/github/droidkaigi/confsched2023/achievements/AchievementsScreenViewModel.kt

```diff
@@ -120,13 +121,17 @@ class AchievementsScreenViewModel @Inject constructor(
         )
     }
 
+    private val clickedAchievement = MutableStateFlow<AchievementAnimationState>(AchievementAnimationState.NotAnimating)
```

Comment:
Could you change this name as we change the state? 🙇

Reply:
Sorry for overlooking. I renamed clickedAchievement to achievementAnimationState.